content/handbook/sales/field-operations/customer-success-strategy/cs-monthly-metrics-dashboard.md

```
@@ -0,0 +1,143 @@
+---
+title: "カスタマーサクセス月次メトリクスダッシュボード"
+upstream_path: /handbook/sales/field-operations/customer-success-strategy/cs-monthly-metrics-dashboard/
+upstream_sha: 1e195b58b9f249ff10bd0e705106c320fee86141
+translated_at: "2026-05-12T00:00:00Z"
+translator: claude
+stale: false
+---
+
+## 目次
+
+1. [概要](#overview)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)